views/users/manage-users.handlebars

```diff
@@ -0,0 +1,43 @@
+<div>
+    USERS 
+    <span class="list-group-item-success">Enabled</span> 
+    <span class="list-group-item-danger">Disabled</span>
+    <span>★Admin★</span> 
+    
+    <br />
+    <code id="manage-error"></code>
+    
+    <br />
+    <ol class="list-group list-group-numbered pull-left">
+        {{#each users as |user| }}
+            <span>
+            {{#if isEnabled}}
+             <li class="list-group-item d-flex justify-content-between list-group-item-success">
+                    <h3 class="mt-3"> {{name}} {{#if isAdmin}} ★ {{/if}} 
+                        <button type="button" class="btn btn-danger pull-right delete-btn" data-bs-toggle="tooltip" data-bs-placement="top" title="Remove from DB" data-user={{id}}>Delete</button>
+                        <button type="button" class="btn btn-secondary pull-right disable-btn" data-bs-toggle="tooltip" data-bs-placement="top" title="Revoke Public Access" data-user={{id}}>Disable</button>
+                    </h3>
+            </li>
+            {{/if}}
+            </span>
+        </li>
```

Comment:
this doesn't seem to belong here. is it extra?

Reply:
yes thank you for finding this